mmv1/third_party/terraform/services/workbench/resource_workbench_instance_accelerator_test.go.erb

```diff
@@ -0,0 +1,76 @@
+<% autogen_exception -%>
+package workbench_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+)
+
+func TestAccWorkbenchInstance_empty_accelerator(t *testing.T) {
```

Comment:
Do you mind moving this test to the file [resource_workbench_instance_test.go.erb](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/services/workbench/resource_workbench_instance_test.go.erb)   ? It will be easier to maintain the code if all of the tests for the same resource in the same file.